pylib/Tools/Launcher/ALPS.py

```diff
@@ -38,7 +40,8 @@
 # @param allocate_cmd              Command to use for allocating nodes from the resource manager
 # @param deallocate_cmd            Command to use for deallocating nodes from the resource manager
 # @param dependencies              List of dependencies specified as the build stage name
-# @}
+# @param checkpoint_file           Optional checkpoint file
+ @}
```

Comment:
this line needs to be commented out, it is causing a syntax error. Did you test these changes?

Reply:
think I fixed this.